ignite/pkg/cosmosanalysis/cosmosanalysis.go

```diff
@@ -18,7 +18,7 @@ import (
 
 const (
 	cosmosModulePath     = "github.com/cosmos/cosmos-sdk"
-	tendermintModulePath = "github.com/tendermint/tendermint"
+	tendermintModulePath = "github.com/cometbft/cometbft"
```

Comment:
I can't find where `tendermintModulePath` constant is being used, is it possible that at some point it was removed leaving the const definition behind? In any case it would make some sense to remove it if the intention is to support other Tendermint implementations.

Reply:
we need to check if the dependency is in `go.mod`

https://github.com/ignite/cli/blob/feat%2Fnew-app-wiring/ignite/pkg/cosmosanalysis/cosmosanalysis.go#L197